HSTracker/UIs/DeckManager/EditDeck.swift

```diff
@@ -109,37 +109,60 @@ class EditDeck: NSWindowController, NSComboBoxDataSource, NSComboBoxDelegate {
 
         NSEvent.addLocalMonitorForEventsMatchingMask(.KeyDownMask) { (e) -> NSEvent? in
             let isCmd = e.modifierFlags.contains(.CommandKeyMask)
-            // let isShift = e.modifierFlags.contains(.ShiftKeyMask)
 
-            guard isCmd else { return e }
+            if isCmd {
+                switch e.keyCode {
+                case 6:
+                    self.window!.performClose(nil)
+                    return nil
 
-            switch e.keyCode {
-            case 6:
-                self.window!.performClose(nil)
-                return nil
+                case 3: // cmd-f
+                    self.searchField.selectText(self)
+                    self.searchField.becomeFirstResponder()
+                    return nil
 
-            case 3: // cmd-f
-                self.searchField.selectText(self)
-                self.searchField.becomeFirstResponder()
-                return nil
+                case 1: // cmd-s
+                    self.save(nil)
+                    return nil
 
-            case 1: // cmd-s
-                self.save(nil)
-                return nil
-
-            case 12: // cmd-a
-                // swiftlint:disable line_length
-                if let selected = self.cardsCollectionView.indexPathsForSelectedItems() as? [NSIndexPath],
+                case 12: // cmd-a

```

Comment:
Removing the ⌘-a, but before we merge that removal I gotta ask: 

How does the ⌘-a work in the first place? :D I could never get it to work/trigger for adding cards.. I think it gets "overridden" by the system defaults for "Edit Menu > Select All"

![image](https://cloud.githubusercontent.com/assets/8146126/15801954/c8bb1ada-2a9c-11e6-82ae-11081ec2b313.png)